README.md

```diff
@@ -465,6 +466,24 @@ Authenticating events by verifying signature
 | clientSecret | <code>string</code> | Client secret can be retrieved from the Adobe I/O Console integration |
 | signatureHeaderValue | <code>string</code> | Value of x-adobe-signature header in each POST request to the registered webhook URL |
 
+<a name="EventsCoreAPI+verifyDigitalSignatureForEvent"></a>
+
+### eventsCoreAPI.verifyDigitalSignatureForEvent(event, recipientClientId, signatureOptions) ⇒ <code>boolean</code>
+Authenticating events by verifying digital signature
+
+**Kind**: instance method of [<code>EventsCoreAPI</code>](#EventsCoreAPI)  
+**Returns**: <code>boolean</code> - If signature matches return true else return false  
+
+| Param | Type | Description |
+| --- | --- | --- |
+| event | <code>object</code> | JSON payload delivered to the registered webhook URL |
+| recipientClientId | <code>string</code> | Target recipient client id retrieved from the Adobe I/O Console integration |
+| signatureOptions | <code>map</code> | signatureOptions map of all digital signature header in each POST request to webhook values consisting fields as below :: 
+                          digiSignature1 : Value of digital signature retrieved from the x-adobe-digital-signature1 header
+                          digiSignature2 : Value of digital signature retrieved from the x-adobe-digital-signature2 header
+                          publicKeyUrl1 : Value of public key url retrieved from the x-adobe-public-key1-url header
+                          publicKeyUrl2 : Value of public key url retrieved from the x-adobe-public-key2-url header
```

Comment:
See how we define the `JournalOptions` object in readme today. We can do the same for the `signatureOptions`.